### PR TITLE
docs - pxf jdbc server file-based config

### PR DESCRIPTION
**Pull request by lisakowen**
in this PR:
- add "configure jdbc connector" to pxf configure landing page
- add new page to configure the jdbc connector
- update the jdbc connector "using" page:
     - add SERVER option to CREATE EXTERNAL TABLE DDL
     - split connection-related custom options from other custom options
     - add jdbc-site.xml property equivalents to connection options
     - update the example to include JDBC connector configuration (register new jdbc driver, create jdbc server config), use SERVER in the external table create DDL
	
doc review site links:
- configure landing page - http://docs-lisa-pxf.cfapps.io/6-0/pxf/instcfg_pxf.html
- new jdbc connector configure page - http://docs-lisa-pxf.cfapps.io/6-0/pxf/jdbc_cfg.html
- jdbc using page - http://docs-lisa-pxf.cfapps.io/6-0/pxf/jdbc_pxf.html

### CODE REVIEW

File: gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Configuring the JDBC Connector (Optional)
+---
+
+You can use PXF to access an external SQL database including MySQL, ORACLE, PostgreSQL, and Hive. This topic describes how to configure the PXF JDBC Connector to access these external data sources.
```
Comment (frankgh) [inline gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: Now that the ignite plugin has been removed, we also suggest using the Ignite JDBC jar.

File: gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+PXF provides a template configuration file for the JDBC Connector. This server template configuration file, located in `$PXF_CONF/templates/jdbc-site.xml`, identifies the minimum set of properties that you must configure to use the PXF JDBC Connector.
+
+The properties in the `jdbc-site.xml` server template file follow:
+
+| Property       | Description                                | Value |
+|----------------|--------------------------------------------|-------|
+| jdbc.driver | Class name of the JDBC driver. | The JDBC driver Java class name; for example `org.postgresql.Driver`. |
+| jdbc.url | The URL that the JDBC driver uses to connect to the database. | The database connection URL (database-specific); for example `jdbc:postgresql://phost:pport/pdatabase`. |
+| jdbc.user | The database user name. | Your user name for connecting to the database. |
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: Here an in the next row, I'd replace: Your -> The

(There's no expectation that the reader would use their own credentials here.)

File: gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+<div class="note">When you configure a PXF JDBC server, you specify the external database user credentials to PXF in clear text in a configuration file.</div>
+
+When you configure the PXF JDBC Connector to access an external SQL database, you add at least one named PXF server configuration for the connector. You:
+
+1. Choose a name for the server configuration.
+2. Create the directory `$PXF_CONF/servers/<server_name>`.
+3. Copy the PXF `jdbc-site.xml` template configuration file to the new server directory.
+4. Fill in appropriate values for the properties in the template file.
+5. Add additional properties and values if required for your environment.
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: Add additional -> Add

Comment (dyozie) [reply gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: Also, it's not clear to me that there are additional properties that can be added to the file.  There don't seem to be any other ones listed in our doc.  Should there be?

File: gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+4. Copy the PXF JDBC server template file to the server configuration directory. For example:
+
+    ``` shell
+    gpadmin@gpmaster$ cp $PXF_CONF/templates/jdbc-site.xml $PXF_CONF/servers/pg_user1_testdb/
+    ```
+        
+5. Open the template server configuration file in the editor of your choice, and provide appropriate property values for your environment. For example, if you are configuring access to a PostgreSQL database named `testdb` on a PostgreSQL instance running on the host named `pgserverhost` for the user named `user1`:
+
+    ``` pre
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: maybe change pre -> xml

File: gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb [hunk 5]
```diff
@@ -85,27 +83,35 @@ The specific keywords and values used in the [CREATE EXTERNAL TABLE](../ref_guid
...
+Example JDBC \<custom-option\> connection strings:
+
+``` pre
+&JDBC_DRIVER=org.postgresql.Driver&DB_URL=jdbc:postgresql://gpmaster:5432/pgtestdb&USER=pguser1&PASS=changeme
+&JDBC_DRIVER=com.mysql.jdbc.Driver&DB_URL=jdbc:mysql://mysqlhost:3306/testdb&USER=user1&PASS=changeme
+```
+
+Additional \<custom-option\>s supported by the `Jdbc` profile include:
 
 | Option Name   | Operation | Description
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/jdbc_pxf.html.md.erb]: Are these options that can also be specified in the jdbc-site.xml file?  I'm guessing not, given the table format.

File: gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb (hunk 3 above)
Comment (lisakowen) [reply gpdb-doc/markdown/pxf/jdbc_cfg.html.md.erb]: good point.  this is more applicable to the hadoop-compatible file systems, where the `*-site.xml` files can contain multiple properties.  i'll remove this step.

**Pull request merged**